Source/IntegrationTests/TestFiles/LitTests/LitTest/gomodule/multimodule/go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/ShubhamChaturvedi7/DafnyRuntimeGo v0.0.0-20231204230030-1d44519b5706 h1:XJBVGl7oX45/PlMpRCdOrp+z8xEtgTMsstmnmG1B/Js=
```

Comment:
Should your fork still appear here given we have DafnyRuntimeGo-gomod now?

Reply:
This is just a metadata for go tooling. I think go maintains a checksum for every single thing ever downloaded, and not really supposed to be edited by hand. I can refresh it but this file is internal to go tooling.

Reply:
I would refresh it unless it's a huge pain. Otherwise the URL to a personal fork is just going to raise red flags for maintainers in the future when they come across it, just as when you accidentally check in references to absolute paths from your laptop with your name in it. :)

Reply:
Did `go mod tidy` but it ended up making things worse. So hand edited them since it had only my repo, and tests still work.